Katello/katello
# Fixes #27395 - Missing CentOS env groups in content view versions

Adds support for copying over environment groups in yum repos.

Note that this only copies over the environment groups.  The environment group information is still not kept track of in Katello.  The environment groups don't appear in the UI and are not installable by package actions.  This implementation should be introduced in a following PR.

To test the patch:

1) Sync CentOS 7 repo http://mirror.centos.org/centos/7/os/x86_64/
2) Add that repo to a content view and publish
3) Subscribe a katello client to that content view and enable the synced repo
4) On the client run 
```yum --disablerepo="*" --enablerepo="<your repo id>" grouplist```

You should see environment groups
```
...
Available Environment Groups:
   Minimal Install
   Compute Node
   Infrastructure Server
   File and Print Server
   Basic Web Server
   Virtualization Host
   Server with GUI
   GNOME Desktop
   KDE Plasma Workspaces
   Development and Creative Workstation
...
```

Alternatively, you could check the repodata/<characters>-comps.xml file in your content view's pulp repo for the environments.
